src/status_im/data_store/realm/schemas/account/v6/contact.cljs

```diff
@@ -37,7 +37,7 @@
   (let [new-contacts (.objects new-realm "contact")]
     (dotimes [i (.-length new-contacts)]
       (let [contact (aget new-contacts i)
-            id      (aget contact "whisper-identity")]
+            id      (.-whisper-identity contact)]
```

Comment:
I am skeptical here. Is it valid JavaScript?

Reply:
why here in particular ?

Reply:
underscore in JavaScript property names are an issue I believe

Reply:
So it should also be a problem for all the other properties with middlescore ?

Reply:
Right

Reply:
@jeluard @yenda Please check this Pastebin with my experiments on the question! I am very surprised how did it work before...
https://pastebin.com/JvhjJZNq

Reply:
@poiga WDYT is the safe option?

Reply:
@pablodip I think you had some thoughts on that point?

Reply:
It seems that dashes [are not supported](https://stackoverflow.com/questions/5516106/are-dashes-allowed-in-javascript-property-names) in property names in Javascript. Not sure if ClojureScript converts property names to strings in those cases.

The safest and clearer option would probably be to access properties with dashes through `goog.object`.

Reply:
I understand. but how did this `(aget contact "whisper-identity")` even work before?

Reply:
`aget` is supposed to work, it's using `object["field"]` JS semantic (not `object.field`)

Reply:
ClojureScript doesn't convert property names, it just silently returns nil.
```
(set! js/foo (js-obj "foo-bar" 1))
(aget js/foo "foo-bar") ;; => 1
(.-foo-bar js/foo) ;; => nil
```

`goog.object/get` should be used instead.